src/plugins/intel_gna/gna2_model_export_helper.cpp

```diff
@@ -68,17 +71,98 @@ TlvFloatRecord GetFloatInTLV(Gna2TlvType type, float value) {
     reinterpret_cast<Gna2TlvRecord*>(r.data())->length = sizeof(float);
     *reinterpret_cast<float*>(r.data() + sizeof(Gna2TlvRecord)) = value;
     return r;
+}  // namespace
+
+std::vector<char> GetStringAsTlv(Gna2TlvType type, const std::string& s) {
+    std::vector<char> record(sizeof(Gna2TlvRecord));
+    reinterpret_cast<Gna2TlvRecord*>(record.data())->type = type;
+
+    std::vector<char> vs(s.begin(), s.end());
+    vs.resize(vs.size() + (4 - vs.size() % 4) % 4, 0);
+    reinterpret_cast<Gna2TlvRecord*>(record.data())->length = vs.size();
+    record.insert(record.end(), vs.begin(), vs.end());
+    return record;
+}
+}  // namespace
+
+#define Gna2DeviceVersionEmbedded3_6 static_cast<Gna2DeviceVersion>(Gna2DeviceVersionEmbedded3_5 + 0x10)
```

Comment:
shouldn't Gna2DeviceVersionEmbedded3_6 be part of enum Gna2DeviceVersion

Reply:
will remove

Reply:
done